content/departments/engineering/submit-pr-faq.md

```diff
@@ -0,0 +1,37 @@
+# Submitting an item to the roadmap
+
+We submit proposals in the form of Amazon-style PR-FAQs. Anybody can submit a PR-FAQ. PR-FAQ can come from the exec team, or from any teammate. PR-FAQ are reviewed and prioritised by the VPEng/Prod with input from the exec team. PMs help build, refine and evaluate the impact of PR-FAQs.
+
+## How to submit a PR-FAQ
+
+1. **Write a PR-FAQ to describe your proposal**. You can use this [template](TODO). If you need help putting one together, post in #product to request help from a PM.
+2. **Get a PM review**. Raise a [PR-FAQ issue](TODO), linking to your proposal. This will trigger an alert in #product and #engineering. Steve will assign a PM (or self-assign for non-customer facing/platform proposals) the issue to:
+   1. Give a pre-read to the PR-FAQ and give (non-blocking) feedback
+   2. Determine the impact of the proposal
+   3. Set a Success Criteria for the project
+3. **Steve will prioritise proposals.** Quarterly, Steve and the exec team will review proposals, with input from PMs (for features) and EMs(for tech-only) to decide what to move to the roadmap.
+4. **Job fair.** Projects moved to the roadmap will be submitted to the job fair.
+
+## Success Criteria
+
+A good success criteria should include:
+
+- a leading indicator of success that can be measured in less than 3 months
+- lagging indicators of success, such as customer adoption that typically takes longer to see
+
+## PM review
+
+PMs (or Steve for non-customer facing/platform proposals) are in charge of reviewing PR-FAQs. This is non-blocking and just a way to ensure there's a high standard for PR-FAQ and that the exec team can focus on reviewing the most impactful proposals first in case there are too many of them.
+
+We don't use a formal/complicated framework for evaluating impact for now, in order to keep things lightweight. We might change introduce a more complicated [opportunity canvas](https://docs.google.com/document/d/1pTEMcwH10xWilQEnVc65oC6PdC3VMjn2XoARfNTaHkc/edit#) later on.
+
+That said, here’s a rough guideline for evaluating impact of customer-facing features:
+
+- **High**: Large ARR (>$5M ARR) or MAU (> 500 DAUs) impact, in the next 12 months. Fit with strategy. Well validated. Differentiated feature. Eg. a candidate for Batch Changes, Insights, Own or adding support for a broadly-used codehost.
+- **Medium**: Large ARR (>$1M ARR) or MAU (> 100 DAUs) impact, in the next 12 months. Fit with strategy. Elements of validation.
+- **Low**: Unknown or unvalidated impact. No clear fit with strategy.
+
+## PR-FAQ tips
+
+- Try to keep the PR section no longer than 2 pages. FAQs can be longer.
+- Try to avoid duplicate proposals or proposals about work already on some team’s roadmap. It’s a good idea to quickly survey what’s on team’s radars before raising a proposal.
```

Comment:
the mentions of "teams" here – I thought we won't have teams with the job fair model. Does this still make sense? Do you just mean "already related to our strategy" or similar?